frontend/src/components/ui/AppDocumentUpload.vue

```diff
@@ -170,4 +170,9 @@ export default {
 .addFileButton {
   font-size: 16px;
 }
+
+:deep(.v-field__input) {
```

Comment:
what is this change for? I tried uploading a file with long name but I don't see any difference.
![image](https://github.com/user-attachments/assets/a05d5768-d4d9-4a27-8f36-51afcd67c9cb)